helm_deploy/values-dev.yaml

```diff
@@ -25,6 +25,7 @@ generic-service:
     API_BASE_URL_PRISON_PERSON: https://prison-person-api-dev.prison.service.justice.gov.uk
     API_BASE_URL_CONTACT_PERSON: https://contacts-api-dev.hmpps.service.justice.gov.uk
     FEATURE_EVENT_COURT_EVENT_CHARGES-INSERTED: false
+    FEATURE_EVENT_PRISONPERSON_PRISON-OFFENDER-EVENTS_PRISONER_BOOKING_MOVED: false
```

Comment:
I think this means the event is still consumed by alerts but not by prison person.